contrib/chatops/actions/format_execution_result.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, config=None, action_service=None):
         api_url = os.environ.get('ST2_ACTION_API_URL', None)
         token = os.environ.get('ST2_ACTION_AUTH_TOKEN', None)
         self.client = Client(api_url=api_url, token=token)
-        self.jinja = jinja_utils.get_jinja_environment()
+        self.jinja = jinja_utils.get_jinja_environment(allow_undefined=True)
```

Comment:
As mentioned in Slack earlier, I have some reservations about this, but it should be safe and might even lead to better UX. Shouldn't be a problem.